docs: improve getting started experience (#6143)
hasura/graphql-engine: docs/graphql/cloud/getting-started/index.rst

  :alt: Hasura Console: Pro tab
  :width: 1000px

Create a table
^^^^^^^^^^^^^^

On the Hasura console, navigate to ``Data -> Create table`` and create a sample table called ``profile`` with
the following columns:

.. code-block:: sql

  profile (
    id SERIAL PRIMARY KEY, -- serial -> auto-incrementing integer
    name TEXT
  )

.. thumbnail:: /img/graphql/core/getting-started/create-profile-table.png
   :alt: Create a table

> Now, insert some sample data into the table using the ``Insert Row`` tab of the ``profile`` table.

Reviewer: ```suggestion
Now, insert some data into the table by clicking on the ``Insert Row`` tab of the ``profile`` table.
```

Try out a query
^^^^^^^^^^^^^^^

Head to the ``GraphiQL`` tab in the console and try running the following query:

.. code-block:: graphql

    query {
      profile {
        id
        name
      }
    }

> You'll see that you get all the inserted data!

Reviewer: ```suggestion
After trying out the query, you will get all the inserted data!
```

.. thumbnail:: /img/graphql/core/getting-started/profile-query.png
   :alt: Try out a query

Next steps
----------

Learn course
^^^^^^^^^^^^

For a full hands-on tour of Hasura, check out our `30-Minute Hasura Basics Course <https://hasura.io/learn/graphql/hasura/introduction/>`__.

Database modelling
^^^^^^^^^^^^^^^^^^

Check out the :ref:`schema section <schema>` to learn about how to model your database schema, as well as how to extend it.

Querying data
^^^^^^^^^^^^^

Learn how to use :ref:`GraphQL queries <queries>` to query data from your GraphQL API.

Inserting data
^^^^^^^^^^^^^^

Learn how to insert data into your GraphQL API using :ref:`GraphQL mutations <mutations>`.

Business logic
^^^^^^^^^^^^^^

There are several options for the implementation of business logic, depending on your use case.

Actions

> *******

Reviewer: Let's also use this here: ^^^^^^^^^^^^^^

:ref:`Actions <actions>` can be used if you'd like to extend your GraphQL schema by integrating with a REST endpoint.

Remote schema

> *************

Reviewer: Let's also use this here: ^^^^^^^^^^^^^^ and the others 🙂
